src/frontend/src/lib/components/icons/OisyWalletLogo.svelte

```diff
@@ -1,51 +1,27 @@
-		/>
-		<path
-			d="M120.65 7.93a2.77 2.77 0 1 0 0 5.55 2.77 2.77 0 0 0 0-5.55Zm-2.97 2.78a2.97 2.97 0 1 1 5.93 0 2.97 2.97 0 0 1-5.93 0Z"
-			clip-rule="evenodd"
-		/>
-		<path
-			d="M121.64 9.75c-.22 0-.45.12-.7.34a2.9 2.9 0 0 0-.3.32s.12.14.25.29a2.94 2.94 0 0 1 .3-.32c.22-.2.37-.25.45-.25a.58.58 0 0 1 0 1.15.3.3 0 0 1-.05 0 .7.7 0 0 0 .28.07c.58 0 .69-.39.7-.42a.96.96 0 0 0-.93-1.18Zm-2 1.91c.21 0 .45-.11.7-.34a2.91 2.91 0 0 0 .3-.32s-.12-.14-.26-.28a2.95 2.95 0 0 1-.3.32c-.22.2-.36.24-.45.24a.57.57 0 0 1-.56-.57c0-.32.25-.58.56-.58a.3.3 0 0 1 .06 0 .7.7 0 0 0-.28-.07c-.58 0-.7.4-.7.42a.96.96 0 0 0 .92 1.18Z"
-		/>
-		<path
-			d="M119.4 10.07c.3.01.6.25.67.31l.31.33.26.3c.28.3.64.65 1 .65a.95.95 0 0 0 .93-.73c-.01.03-.16.42-.7.4-.3 0-.6-.24-.66-.3-.17-.16-.32-.33-.32-.33l-.25-.29c-.28-.31-.64-.66-1-.66a.95.95 0 0 0-.92.73c0-.05.16-.42.69-.4Zm-.69.41Z"
+			fill-rule="evenodd"
 			clip-rule="evenodd"
+			d="M24.412 33.4193L23.8827 35.7603C23.8037 36.1099 23.4693 36.3405 23.1152 36.2858C22.308 36.161 21.49 36.0177 20.6637 35.8594C20.2842 35.7867 20.0423 35.4133 20.1275 35.0365L20.6067 32.917C20.2711 32.8513 19.9334 32.7802 19.594 32.7042C19.288 32.6344 18.985 32.5615 18.6854 32.4855L18.2064 34.6044C18.1212 34.981 17.7426 35.214 17.3689 35.1168C16.5548 34.905 15.7547 34.6836 14.9723 34.4504C14.6284 34.3479 14.4251 33.9955 14.5043 33.6455L15.0308 31.3164C9.82695 29.2297 6.54161 25.683 7.93603 19.6262L8.27156 18.1421C9.62094 12.0626 14.1109 10.2811 19.7056 10.6393L20.2318 8.31177C20.3109 7.96173 20.646 7.73105 21.0006 7.78645C21.8072 7.91249 22.6247 8.05683 23.4507 8.21592C23.8299 8.28895 24.0714 8.66213 23.9863 9.03875L23.5073 11.1575C23.8104 11.2179 24.1153 11.2824 24.4215 11.3511C24.7609 11.4278 25.0965 11.5083 25.4277 11.5927L25.9065 9.4751C25.9917 9.09826 26.3707 8.86528 26.7445 8.96289C27.5586 9.17544 28.3586 9.39788 29.141 9.63241C29.4843 9.73534 29.6869 10.0874 29.6079 10.437L29.0801 12.7717C34.2161 14.858 37.4261 18.402 36.0643 24.4256L35.7288 25.9097C34.3909 31.9376 29.9492 33.7421 24.412 33.4193ZM31.9629 24.7039L32.1517 23.8687C33.4814 18.3434 29.0601 16.2797 23.6094 14.9676C18.0988 13.8015 13.2196 13.7626 12.0432 19.3226L11.8544 20.1578C10.5189 25.7084 14.9403 27.7721 20.4165 29.09C25.9016 30.2503 30.7807 30.2893 31.9629 24.7039Z"
+			fill="white"
 		/>
-	</g>
-	<defs>
-		<linearGradient id="a" x1="16.93" x2="16.93" y1="36.57" y2="0" gradientUnits="userSpaceOnUse">
-			<stop stop-color="#00CADD" />
-			<stop offset="1" stop-color="#016DFC" />
-		</linearGradient>
-		<linearGradient
-			id="b"
-			x1="26.1"
-			x2="10.32"
-			y1="15.94"
-			y2="15.94"
-			gradientUnits="userSpaceOnUse"
-		>
-			<stop stop-color="#02E9C6" />
-			<stop offset="1" stop-color="#02E9C6" stop-opacity="0" />
-		</linearGradient>
-	</defs>
-</svg>
+	</svg>
+</div>
+
+<picture>
```

Comment:
should the picture have an accessbility hint?

Reply:
good point! one of the two has it, but the other no: corrected!

Reply:
Not sure what you mean, it's fine or you gonna push something?

Reply:
now pushed! (i thought i had done it...)